packages/additional-materials/src/moderation/generateAdditionalMaterialModeration.ts

```diff
@@ -20,5 +23,17 @@ export const generateAdditionalMaterialModeration = async ({
     provider,
   );
 
-  return moderation;
+  // Transform the LLM response format to the internal ModerationResult format
+  // LLM returns: { scores, justifications, flagged_categories }
+  // Internal expects: { scores?, categories, justification? }
+  const result: ModerationResult = {
```

Comment:
Wondering if makes sense to version what gets persisted here rather than shoehorn it into the previous format?

Reply:
Looks like the idea is to make it backwards compatible to minimise required changes but is that just temporary?